### ISSUE TITLE
Preact port of the new react-transition-group ?

### ISSUE THREAD

**Issue by theochampion** [developit/preact-transition-group#8]
Hi there, love what you are doing with Preact, it's awesome ! ;)

I've been searching for the equivalent of the new https://github.com/reactjs/react-transition-group in Preact.

Is this what this is supposed to be ? If yes what is the equivalent of the CSSTransition component ?

Thanks !

**Comment by vinaybedre** [comment]
As I see, there is no CSSTransition class exported in preact-transition-group

**Comment by andrewiggins** [comment]
@theochampion @vinaybedre FYI, I've begun a rewrite of this package based on the new v2 react-transition-group which includes the `CSSTransition` component. A PR should be out in the (hopefully) near future.

**Comment by lchimaru** [comment]
@andrewiggins any progress?

**Comment by andrewiggins** [comment]
Yes! I've gotten a good amount of the work done in my [andrewiggins/preact-transition-group refork-v2 branch](https://github.com/andrewiggins/preact-transition-group/tree/refork-v2). Still working through fixing all the tests.

**Comment by shoojx** [comment]
@andrewiggins any chance you can port this over to preact-x? thanks!

**Comment by lohfu** [comment]
@andrewiggins i have converted the latest react-transition-group to work with preact X. it wasn't that much work, but i've struggled with a couple of the tests. 13 out of 54 still fail. it's fully functional now as far as i can tell, but i think i'm gonna need some help bringing it over the finish line. if you guys are interested that is.

i've pushed my changes to the preact branch in a forked repo, https://github.com/lohfu/react-transition-group/tree/preact

regarding the tests, i think most of problems stem from inconsistencies in the react and preact adapters for enzyme, but i'm not 100% sure. i bet if you have a look you'll solve most of them in a jiffy.
